oshi-core-java25/src/main/java/oshi/ffm/mac/MacSystemFunctions.java

```diff
@@ -23,18 +24,20 @@ public final class MacSystemFunctions extends ForeignFunctions {
     private MacSystemFunctions() {
     }
 
+    private static final SymbolLookup SYSTEMB = libraryLookup("System");
```

Comment:
I am not sure if there is some more lifecycle management necessary for loading a library with ffm, though the tests seem to work with that.